Runtime/Core/ApiGatewayManagement/AnywhereGameServerAdapter.cs

```diff
@@ -0,0 +1,118 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.GameLift;
+using AmazonGameLiftPlugin.Core.ApiGatewayManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared;
+using AmazonGameLiftPlugin.Core.Shared.Logging;
+using Amazon.GameLift.Model;
+
+namespace AmazonGameLiftPlugin.Core.ApiGatewayManagement 
+{
+    public class AnywhereGameServerAdapter : IGameServerAdapter
```

Comment:
We should add unit tests for this file (like we have similar unit tests for the GameLiftLocal adapter that we plan to remove